docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,2176 @@
+
+![Relative Floorplan](./images/core_util.png)
+
+### IO Pin Placement
+
+Place pins on the boundary of the die on the track grid to minimize net
+wirelengths. Pin placement also creates a metal shape for each pin using
+min-area rules.
+
+For designs with unplaced cells, the net wirelength is computed considering
+the center of the die area as the unplaced cells position.
+
+Find pin placement document [here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ppl/README.md).
+
+Refer to the built-in examples [here](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/ppl/test).
+
+Launch openroad GUI by running the following command(s) in the terminal in OpenROAD tool root directory:
+
+```
+cd src/pad/test/
+openroad -gui
+```
+
+Run [place_pin4.tcl](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ppl/test/place_pin4.tcl) script to view
+pin placement.
+
+From the GUI `Tcl commands` section:
+
+```
+source place_pin4.tcl
+```
+
+View the resulting pin placement in GUI:
+
+![place_pin](./images/place_pin.webp)
+
+In OpenROAD GUI to enlarge `clk` pin placement, hold mouse right button
+as follows and draw sqaure box in specific location:
+
+![pin_zoom](./images/pin_zoom_RC.webp)
+
+Now `clk` pin zoom to clear view as follows:
+
+![pin_zoomed](./images/pin_zoomed.webp)
+
+
+### Chip Level IO Pad Placement
+
+In this section, you will generate an I/O pad ring for the `coyote` design
+using a Tcl script.
+
+ICeWall is a utility to place IO cells around the periphery of a design,
+and associate the IO cells with those present in the netlist of the
+design.
+
+For I/O pad placement using ICeWall refer to the readme file
+[here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/pad/README.md).
+
+Refer to the built-in examples [here](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/pad/test).
+
+Launch openroad GUI by running the following command(s) in the terminal in OpenROAD tool root directory:
+
+```
+cd src/pad/test/
+openroad -gui
+```
+
+Run [coyote_tc_sky130.tcl](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/pad/test/coyote_tc_sky130.tcl) script
+to view IO pad placement.
+
+From the GUI `Tcl commands` section:
+
+```
+source coyote_tc_sky130.tcl
+```
+
+View the resulting IO pad ring in GUI:
+
+![coyote pad ring](./images/coyote_pad_ring.png)
+
+### Power Planning And Analysis
+
+In this section, you will use the design `gcd` to create a
+power grid and run power analysis.
+
+Pdngen is used for power planning. Find the document [here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/pdn/README.md).
+
+Refer to the built-in examples [here](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/pdn/test).
+
+Launch openroad GUI by running the following command(s) in the terminal in OpenROAD tool root directory:
+
+```
+cd src/pdn/test
+openroad -gui
+```
+
+<!-- Run [test_gcd.api.tcl](../../src/pdn/test/test_gcd.api.tcl)
```

Comment:
@vvbandeira this script is not there anymore as well

Reply:
same update required.